<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkMultiLabelManagerControls</class>
 <widget class="QWidget" name="QmitkMultiLabelManagerControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>69</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QToolButton" name="btnAddLabel">
       <property name="toolTip">
        <string>Add a new label to the group of the selected label.</string>
       </property>
       <property name="text">
        <string>L+</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveLabel">
       <property name="toolTip">
        <string>Deletes the selected label (all instances).</string>
       </property>
       <property name="text">
        <string>L-</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnAddInstance">
       <property name="toolTip">
        <string>Add a new instance of the selected label.</string>
       </property>
       <property name="text">
        <string>I+</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveInstance">
       <property name="toolTip">
        <string>Deletes the selected label instance.</string>
       </property>
       <property name="text">
        <string>I-</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnAddGroup">
       <property name="toolTip">
        <string>Add a new group.</string>
       </property>
       <property name="text">
        <string>G+</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveGroup">
       <property name="toolTip">
        <string>Remove the current group (including all labels).</string>
       </property>
       <property name="text">
        <string>G-</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnSavePreset">
       <property name="toolTip">
        <string>Save the labels preset.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnLoadPreset">
       <property name="toolTip">
        <string>Load a label preset from file.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QmitkMultiLabelInspector" name="labelInspector" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkSearchBox" name="labelSearchBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkSearchBox</class>
   <extends>QLineEdit</extends>
   <header location="global">ctkSearchBox.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkMultiLabelInspector</class>
   <extends>QWidget</extends>
   <header>QmitkMultiLabelInspector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
